pkg/framework/host.go

```diff
@@ -222,20 +242,20 @@ func (h *Host) PodName(namespace, labelSelector string) (string, error) {
 	return pod.Name, nil
 }
 
-func (h *Host) Setup() error {
+func (h *Host) Setup(b *backoff.ExponentialBackOff) error {
```

Comment:
I would just have made the backoff configurable per `Host` type initialization with the config structure. Then the interfaces do not change. I think in almost all cases we do not need to configure a different backoff per framework per action.

Reply:
Sure, done PTAL.